specification/postgresql/resource-manager/readme.md

```diff
@@ -39,11 +39,31 @@ tag: package-flexibleserver-2021-06
 tag: package-2020-01-01
 ```
 
+
+### Tag: package-flexibleserver-2022-03-preview
+
+These settings apply only when `--tag=package-flexibleserver-2022-03-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-flexibleserver-2022-03-preview'
+input-file:
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Administrators.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Backups.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Capabilities.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/CheckNameAvailability.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Configuration.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Databases.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/FirewallRules.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/ServerStartStopRestart.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/VirtualNetwork.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/PrivateDnsZone.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/postgresql.json
+  - Microsoft.DBforPostgreSQL/preview/2022-03-08-privatepreview/getCachedServerName.json
+```
 ### Tag: package-flexibleserver-2022-03-privatepreview
 
 These settings apply only when `--tag=package-flexibleserver-2022-03-privatepreview` is specified on the command line.
 
-``` yaml $(tag) == 'flexibleserver-2022-03-privatepreview'
+``` yaml $(tag) == 'package-flexibleserver-2022-03-privatepreview'
```

Comment:
Could LintDiff issue be caused by this similar name..? In that case, we can remove 2022-03-privatepreview..

Reply:
Tried but error still remains

Reply:
Resolved with my commit